### PR TITLE
Added DX_DATE column to the diagnosis table

### PR DESCRIPTION
**Pull request by schandaka**

### CODE REVIEW

File: Oracle/diagnosis.sql [hunk 1]
```diff
@@ -12,6 +12,7 @@ CREATE TABLE diagnosis(
 	ENCOUNTERID varchar(50) NOT NULL,
 	ENC_TYPE varchar(2) NULL,
 	ADMIT_DATE date NULL,
+    DX_DATE date NULL,
```
Comment (dckc) [inline Oracle/diagnosis.sql]: any particular reason to indent this differently?

Do the style guidelines in CONTRIBUTING.md need updating?

File: Oracle/diagnosis.sql [hunk 2]
```diff
@@ -177,7 +178,7 @@ insert into poafact
 execute immediate 'create index poafact_idx on poafact (patient_num, encounter_num, provider_id, concept_cd, start_date)';
 GATHER_TABLE_STATS('POAFACT');
 
-insert into diagnosis (patid, encounterid, enc_type, admit_date, providerid, dx, dx_type, dx_source, dx_origin, pdx, dx_poa, raw_dx_poa)
+insert into diagnosis (patid, encounterid, enc_type, admit_date,dx_date, providerid, dx, dx_type, dx_source, dx_origin, pdx, dx_poa, raw_dx_poa)
```
Comment (dckc) [inline Oracle/diagnosis.sql]: missing space after comma

**Review by dckc — APPROVED** [review]
I'd appreciate consistent formatting, but it's not critical.

File: Oracle/diagnosis.sql (hunk 1 above)
Comment (schandaka) [reply Oracle/diagnosis.sql]: changed the indent. for some reason in sql developer tab was getting converted to 4 spaces. not sure why.

**Pull request merged**